script.js

```diff
@@ -1,4 +1,77 @@
+// assign numbers to scissors (1), paper (2), stone (3)
+// get user input
+// get computer's play
+// convert user input into integer
+// if user input == computer play, it's a draw
+// if user input == computer play - 1 OR + 2, user wins
+// else, computer wins
+
+var winCount = 0;
+
+var randomNumberGenerator = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal) + 1;
+  return randomInteger;
+};
+
+var convertSPSstringToInteger = function (SPSstring) {
+  if (SPSstring == "scissors") return 1;
+  if (SPSstring == "paper") return 2;
+  if (SPSstring == "stone") return 3;
+  return 4;
+};
+
+var userName = "";
+
+var convertSPSIntegerToString = function (SPSInteger) {
+  if (SPSInteger == 1) return "scissors";
+  if (SPSInteger == 2) return "paper";
+  if (SPSInteger == 3) return "stone";
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  var computerSPSInteger = randomNumberGenerator();
+  var computerSPSstring = convertSPSIntegerToString(computerSPSInteger);
+  console.log(computerSPSstring);
+  var playerSPSinteger = convertSPSstringToInteger(input);
```

Comment:
Seems overly complicated to do so many conversions. 

Would be simpler if we did the following 
1. Make a function that returns either scissors paper or stone. e.g. a function called `generateRandomSPS`
2. Make a variable to get this random value.  e.g. var computerHand = `generateRandomSPS()`
3. Compare the userInput  and computerHand